google/location.go

```diff
@@ -29,8 +29,15 @@ func (l *Location) Close() error {
 func (l *Location) CreateContainer(containerName string) (stow.Container, error) {
 
 	projId, _ := l.config.Config(ConfigProjectId)
+	location, _ := l.config.Config(ConfigLocation)
+	storageClass, _ := l.config.Config(ConfigStorageClass)
+
 	// Create a bucket.
-	_, err := l.client.Buckets.Insert(projId, &storage.Bucket{Name: containerName}).Do()
```

Comment:
This code is quite out of date (1+ year) vs greymata, maybe we should update but it can be a different PR https://github.com/graymeta/stow/pull/248